bridges/BakaUpdatesMangaReleasesBridge.php

```diff
@@ -0,0 +1,84 @@
+<?php
+class BakaUpdatesMangaReleasesBridge extends BridgeAbstract {
+	const NAME = 'Baka Updates Manga Releases';
+	const URI = 'https://www.mangaupdates.com/';
+	const DESCRIPTION = 'Get the latest series releases';
+	const MAINTAINER = 'fulmeek';
+	const PARAMETERS = array(array(
+		'series_id' => array(
+			'name'			=> 'Series ID',
+			'exampleValue'	=> '12345'
+		)
+	));
+	const LIMIT_ITEMS = 10;
+
+	private $feedName = '';
+
+	public function collectData() {
+		$series_id = preg_replace('/[^0-9]/', '', $this->getInput('series_id'));
```

Comment:
Instead of parsing the value manually, specify the correct format in the `PARAMETERS` array: https://github.com/RSS-Bridge/rss-bridge/wiki/const-PARAMETERS#level-2---parameter

Since a numeric input is expected, either use the 'number' type (recommended) or add a pattern.

Reply:
That's probably a better option, thanks.